SmartDeviceLink/SDLConfiguration.m

```diff
@@ -59,6 +59,7 @@ - (instancetype)initWithLifecycle:(SDLLifecycleConfiguration *)lifecycleConfig l
     _lifecycleConfig = lifecycleConfig;
     _lockScreenConfig = lockScreenConfig ?: [SDLLockScreenConfiguration enabledConfiguration];
     _loggingConfig = logConfig ?: [SDLLogConfiguration defaultConfiguration];
+    _streamingMediaConfig = streamingMediaConfig;
 
     if (_streamingMediaConfig != nil) {
         NSAssert(!([_lifecycleConfig.appType isEqualToEnum:SDLAppHMITypeNavigation] || [_lifecycleConfig.appType isEqualToEnum:SDLAppHMITypeProjection]), @"You should only set a streaming media configuration if your app is a NAVIGATION or PROJECTION HMI type");
```

Comment:
This assert statement is getting called when app type *is* set to Navigation. Since assert statements only get called if they are false, the **!** should be removed from the conditional statement.